packages/core/database/src/migrations/internal-migrations/5.0.0-04-published-at.ts

```diff
@@ -11,7 +11,7 @@ import type { Migration } from '../common';
  */
 const createPublishedAtColumn = async (db: Knex, tableName: string) => {
   await db.schema.alterTable(tableName, (table) => {
-    table.string('published_at');
+    table.string('published_at').defaultTo(new Date().getTime());
```

Comment:
I am not completely sure this is how I should format a date, @alexandrebodin  do you remember if we format it differently for each db?

Reply:
you should be ok passing in a new Date()  directly